remotevbox/vbox.py

```diff
@@ -13,7 +13,7 @@
 
 
 class IVirtualBox(object):
-    def __init__(self, location, user, password):
+    def __init__(self, location, user="", password=""):
```

Comment:
Can Virtualbox Webservice really work without specifying user/password or with empty ones? I would be happy to know how to do it then 😮